src/core/ComponentCell/style.ts

```diff
@@ -0,0 +1,31 @@
+import { styled } from "@mui/material/styles";
+import { CommonThemeProps, getSpaces } from "../styles";
+
+const contentPositionMapping = {
```

Comment:
Nicely done 💯

Reply:
@masoudmanson should this be called ContentPositionMapping 🤔